rest-server/core/src/main/scala/com/actionml/core/engine/LambdaAlgorithm.scala

```diff
@@ -30,7 +30,7 @@ trait LambdaAlgorithm[T] extends LazyLogging {
     * @param datum The input Event type defined by the parmeterized type
     * @return
     */
-  def input(datum: T): Validated[ValidateError, Boolean] = Valid(true)
+  def input(datum: T): Validated[ValidateError, String] = Valid("")
```

Comment:
Seems like we should 

- change all `Validated[ValidateError, Boolean]` cases to `Validated[ValidateError, String]`
- change ALL Strings returned to JSON